dockerfiles/Dockerfile.oracle

```diff
@@ -0,0 +1,74 @@
+FROM ghcr.io/oracle/oraclelinux:8-slim as builder-base
```

Comment:
You need to check with Bhat to see if the new common base image can/should be used.

Reply:
Will check